pyproject.toml

```diff
@@ -44,6 +44,8 @@ pytest-mock = "^1.11.0"
 pytest-asyncio = "^0.10.0"
 pytest-xdist = "^1.30"
 typing_extensions = "^3.7"
+flask = "^1.1"
+werkzeug = "^0.16.0"
```

Comment:
Most probably we will need `werkzeug` in common dependencies, not only in dev ones